ProjFS.Mac/PrjFSLib/prjfs-log/prjfs-log.cpp

```diff
@@ -24,7 +24,9 @@ int main(int argc, const char * argv[])
         std::cerr << "Failed to set up shared data queue.\n";
         return 1;
     }
-
+    
+    __block int lineCount = 0;
```

Comment:
Is the `__block` required here because of something with Objective-C?

Reply:
Not Objective-C per se. Apple's C++ compiler has an extension to support blocks, which is how we can call the `dispatch_source_set_event_handler` function down below. This `__block` flag is required to indicate that you want to be able to modify a variable inside a block.

Reply:
Ahh okay thanks! I missed that `dispatch_source_set_event_handler` was in a block.